### PR TITLE
Designing New Splash Screen

### LINKED ISSUE #66: Design a splash screen for app
Currently app doesnot have a splash screen . We can add splash screen to the app .
Would you like to work on this issue?
Yes

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #66 
New Splash screen of app designed as!
image

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@imjog @chashmeetsingh please review the pr!

**Review by jogendra — COMMENTED** [review]
Check the failing builds!

**Comment by SIMRAN1** [comment]
@imjog can you please tell me why is the build failing??

**Comment by jogendra** [comment]
@SIMRAN1 Tests are failing. Run test in Xcode and see why they are failing.

**Comment by SIMRAN1** [comment]
@imjog I have tested in xcode no test is failing!!

**Comment by SIMRAN1** [comment]
@imjog I got the test failed when I erased all contents of simulator and then run the test!
This error UIRefreshControl.init() must be used from main thread only in ScheduleViewController.swift
Shall I open issue for it?

File: FOSSAsia/ScheduleViewController.swift [hunk 1]
```diff
@@ -34,9 +36,10 @@ class ScheduleViewController: EventsBaseViewController, SwipeToFavoriteCellPrese
         super.viewDidLoad()
 
         viewModel?.delegate = self
-        refreshControl.attributedTitle = NSAttributedString(string: "Pull to refresh")
-        refreshControl.addTarget(self, action: #selector(ScheduleViewController.refreshData(_:)), for: .valueChanged)
-        tableView.addSubview(refreshControl)
+        self.refreshControl = UIRefreshControl()
+        refreshControl!.attributedTitle = NSAttributedString(string: "Pull to refresh")
```
Comment (jogendra) [inline FOSSAsia/ScheduleViewController.swift]: check `refreshControl` with `if let` or `guard let`, to be safe!

Comment (SIMRAN1) [reply FOSSAsia/ScheduleViewController.swift]: okk @imjog but I was thinking to create another pr for resolving failed testcases !!

Comment (jogendra) [reply FOSSAsia/ScheduleViewController.swift]: Do  not create another PR if you resolved it here

Comment (jogendra) [reply FOSSAsia/ScheduleViewController.swift]: @SIMRAN1 Please check the resolved failing tests: #73

Comment (SIMRAN1) [reply FOSSAsia/ScheduleViewController.swift]: okk @imjog I have seen #73 !!

**Comment by SIMRAN1** [comment]
@imjog @chashmeetsingh please review the pr!!

**Review by jogendra — CHANGES_REQUESTED** [review]
- Increase the size of the logo (refer: SUSI iOS)
- Use the good quality of png, current png in not looking good

**Comment by SIMRAN1** [comment]
okk @imjog !!

**Comment by SIMRAN1** [comment]
@imjog  @chashmeetsingh please review and merge the pr!!

**Review by jogendra — CHANGES_REQUESTED** [review]
Please remove the status bar on splash screen. Follow: https://stackoverflow.com/questions/35280747/hide-statusbar-during-splash-screen

**Comment by SIMRAN1** [comment]
okk @imjog !!

**Comment by SIMRAN1** [comment]
Done @imjog !!
image

**Comment by SIMRAN1** [comment]
@imjog @chashmeetsingh please review it!!

**Review by jogendra — CHANGES_REQUESTED** [review]
Bar is fine but the logo on splash screen still look small to me and logo png quality is not good. Please ask on channel if they can provide you better quality logo. Current, quality doesn't look good to me.

**Comment by SIMRAN1** [comment]
Thanks @imjog I will ask in channel for good quality png !!

**Review by jogendra — COMMENTED** [review]
@SIMRAN1 Please finish one PR and then start working on other PR. This PR is made 20+ days ago and still not fulfill the requested changes. I think the issue you are solving in this PR is not a big issue.

**Comment by SIMRAN1** [comment]
image
@imjog I was busy with other PRs so I was not able to modify it previously ...I have done the changes please Review it!

**Comment by SIMRAN1** [comment]
@imjog @jurvis @chashmeetsingh please review the pr!

**Review by jogendra — APPROVED** [review]
Looking good now!

**Comment by SIMRAN1** [comment]
thanks @imjog !

**Pull request merged**